Tasks/ANTV1/task.json

```diff
@@ -16,8 +16,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 140,
-        "Patch": 1
+        "Minor": 141,
+        "Patch": 0
```

Comment:
We shouldn't need to update task version since nothing is changing functionally. @stephenmichaelf : any guidance here?

Reply:
I checked with Stephen. He advised to bump the versions.